packages/consumption/test/modules/requests/testHelpers/TestObjectFactory.ts

```diff
@@ -61,18 +58,13 @@ export class TestObjectFactory {
             cache:
                 properties?.cache ??
                 CachedRelationship.from({
-                    changes: [
+                    auditLog: [
```

Comment:
there is a good amount of cheating going on in this factory - there then also should be a cachedAt date. I also wanted to ask if cache/cachedAt should be set to mandatory - for relationships they are, but this would be inconsistent with other types